openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -76,6 +76,14 @@ instances:
     #
     # send_histograms_buckets: true
 
+    ## @param send_distribution_buckets - boolean - optional - default: false
+    ## Send histograms as datadog distribution metrics.
+    ## This requires send_histograms_buckets to be true.
+    ##
```

Comment:
```suggestion
    ## Set `send_histograms_buckets` to `true` to send histograms as Datadog distribution metrics.
```

Reply:
Should this replace `This requires send_histograms_buckets to be true` or do you want an additional line?

Reply:
Nop it should replace it :)

Reply:
the multiline comment + suggested change does not play well 🤦‍♂